src/test/java/net/pms/io/IOTest.java

```diff
@@ -0,0 +1,140 @@
+package net.pms.io;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.PushbackInputStream;
+import java.io.RandomAccessFile;
+import java.util.ArrayList;
+import java.util.List;
+
+import net.sf.sevenzipjbinding.ExtractOperationResult;
+import net.sf.sevenzipjbinding.IInArchive;
+import net.sf.sevenzipjbinding.ISequentialOutStream;
+import net.sf.sevenzipjbinding.SevenZip;
+import net.sf.sevenzipjbinding.SevenZipException;
+import net.sf.sevenzipjbinding.impl.RandomAccessFileInStream;
+import net.sf.sevenzipjbinding.simple.ISimpleInArchive;
+import net.sf.sevenzipjbinding.simple.ISimpleInArchiveItem;
+
+public class IOTest {
+	public static void main(String[] args) throws Exception {
+		new IOTest().testFileConversion();
+	}
+	
+	public void testFileConversion() throws Exception {
+		// Test file conversion
+		List<String> cmd = new ArrayList<>();
+		cmd.add("bin\\win32\\ffmpeg64.exe");
+		cmd.add("-i");
+		cmd.add("C:\\Users\\IBM_ADMIN\\Downloads\\jungle\\utbr.mp3");
```

Comment:
@Sami32 This seems to be a test class (the name is a hint) and this isn't "finished" code so I doubt that class is supposed to be a part of any released version.